SU2_CFD/include/fluid/CCoolPropConductivity.hpp

```diff
@@ -0,0 +1,80 @@
+/*!
+* \file CCoolPropConductivity.hpp
+* \brief Defines laminar thermal conductivity model from CoolProp.
+* \author P.YAn, G. Gori, A. Guardone
+* \version 7.4.0 "Blackbird"
+*
+* SU2 Project Website: https://su2code.github.io
+*
+* The SU2 Project is maintained by the SU2 Foundation
+* (http://su2foundation.org)
+*
+* Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+*
+* SU2 is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* SU2 is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+
+#include "CConductivityModel.hpp"
+#include "CCoolProp.hpp"
+
+#ifdef USE_COOLPROP
+#include "CoolProp.h"
+#include "AbstractState.h"
+#endif
+/*!
+* \class CCoolPropConductivity
+* \brief Defines conductivity model from CoolProp.
+* \author P.Yan
+*/
+class CCoolPropConductivity final : public CConductivityModel {
+ private:
+#ifdef USE_COOLPROP
+  std::unique_ptr<CoolProp::AbstractState> fluid_entity;   /*!< \brief fluid entity */
+#endif
+  /*!< \brief list of fluids whose conductivity model is available in CoolProp */
+  vector<string> fluidNameList = {"Air", "Ammonia", "Argon", "Benzene", "CarbonDioxide", "Cyclopentane", "EthylBenzene",
+                              "Ethane", "Ethanol", "HeavyWater", "Helium", "Hydrogen", "IsoButane", "Isopentane",
+                              "Methane", "Methanol", "Nitrogen", "Oxygen", "Propylene", "ParaHydrogen", "R11", "R116" , "R12", "R123",
+                              "R1234yf", "R1234ze(E)", "R124", "R125", "R13", "R134a", "R14", "R141b", "R142b", "R143a", "R152A",
+                              "R218", "R22", "R227EA", "R23", "R236EA", "R236FA", "R245fa", "R32", "R404A", "R407C", "R410A",  "R507A",
+                              "RC318",  "SulfurHexafluoride", "Toluene", "Water", "m-Xylene", "n-Butane", "n-Decane", "n-Dodecane", "n-Heptane",
+                              "n-Hexane", "n-Nonane", "n-Octane", "n-Pentane", "n-Propane", "o-Xylene", "p-Xylene"};
```

Comment:
Define this list as a `const static` member of CCoolProp so that you don't have to duplicate it for Conductivity and Viscosity.

Reply:
sorry, but the fluidNameList in these two files are slightly different. 
For example, for fluid R1233zd(E), viscosity model is available but conductivity model is not.